easybuild/easyconfigs/p/ParaView/ParaView-5.5.2-foss-2018b-mpi.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'CMakeMake'
+
+name = 'ParaView'
+version = '5.5.2'
+versionsuffix = '-mpi'
```

Comment:
@hrnagel Since `Python` is a runtime dependency, please indicate this in the `versionsuffix`:

```python
versionsuffix = '-Python-%(pyver)s-mpi'
```

(note that this implies renaming the easyconfig file as well)